node/validator.go

```diff
@@ -56,13 +73,24 @@ func NewValidator(config *ValidatorConfig) Validator {
 		return nil
 	}
 
+	acc, err := account.New(&account.Config{
+		Mode:             config.PayAccountMode,
+		PrivateKey:       config.PrivateKey,
+		KeystorePath:     config.KeystorePath,
+		PasswordFilePath: config.PasswordFilePath,
+		Address:          config.PayAccountAddress})
+	if err != nil {
+		log.Panicw("failed to create account", "err", err)
+	}
+
 	v := &validator{
 		cfg:       config,
 		client:    cli,
 		scheduler: gocron.NewScheduler(time.UTC),
+		account:   acc,
 	}
 
-	if _, err := v.scheduler.Every(1).Hours().Do(func() {
+	if _, err := v.scheduler.Every(30).Second().Do(func() {
```

Comment:
Sentry maybe used in testnet which get less validators. Let us change it to 10 second.

Reply:
done